actions/lib/action.py

```diff
@@ -14,13 +14,9 @@ class NetboxBaseAction(Action):
 
     def __init__(self, config):
         super(NetboxBaseAction, self).__init__(config)
-
-    def get(self, endpoint_uri, **kwargs):
-        """Make a get request to the API URI passed in
-        """
-
-        self.logger.debug("Calling base get with kwargs: {}".format(kwargs))
-
+        
+    def _make_request(self, endpoint_uri, http_action, **kwargs):
```

Comment:
Please add a method doc string to point out that this is a consolidation of the actual logic to make all requests.